src-docs/src/views/window_event/window_event_conflict.js

```diff
@@ -12,50 +12,43 @@ import {
 
 import { ModalExample } from './modal_example_container';
 
-class ConflictModal extends React.Component {
-  constructor(props) {
-    super(props);
+const ConflictModal = props => {
+  const [inputValue, setInput] = useState('');
```

Comment:
Sorry! I just noticed this `setInput` and I think it should be called `setInputValue`.

```suggestion
  const [inputValue, setInputValue] = useState('');
```

Reply:
Changed to `setInputValue`